### DISCUSSION TITLE
[Blog] Reading progress indicator

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#6149] [Ideas]
Could be a nice idea to have a indicator for how far you have been in a blog post. Like a line at the top that slowly becomes fuller the further down you scroll.

**Comment by alexvoss** [comment]
One thing to make sure is to not confuse readers with the [progress indicator for the instant navigation plugin](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation#progress-indicator) used at the same time. It would certainly be possible and relatively simple, I think, to add this as a customization but perhaps this does make a useful feature.

**Reply by squidfunk** [reply]
Since you'd only have to track scroll progress, it doesn't need any support from the theme, so customization is the way to go. For example, here's an implementation you can add via [additional JavaScript](https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript):

https://dev.to/leonardoschmittk/how-to-make-a-scroll-indicator-bar-with-js-html-and-css-easily-and-explained-2jbg
